src/seedu/addressbook/data/person/Person.java

```diff
@@ -33,6 +36,10 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
     public Person(ReadOnlyPerson source) {
         this(source.getName(), source.getPhone(), source.getEmail(), source.getAddress(), source.getTags());
     }
+    
+    public int getSequence(){
```

Comment:
`getSequenceNumber`? Don't deviate from naming conventions unless you have a strong reason.